repos/rm.sh

```diff
@@ -21,7 +21,6 @@ cfengine_rm masterfiles
 cfengine_rm enterprise
 cfengine_rm nova
 cfengine_rm mission-portal
-cfengine_rm design-center
```

Comment:
I would suggest wait with removing this line.
Otherwise, an upgrade procedure to this version should include note to `rm design-center` either manually or via this script. And we don't have upgrade procedure :-)